### DISCUSSION TITLE
Is there a way to restrict image zoom?

### DISCUSSION THREAD

**Discussion by trpx** [advanced-cropper/vue-advanced-cropper#140] [Q&A] [answered]
E.g. I would like to prevent users from zooming (both in and out) too much, is there a way to do this? I failed to find a way to do this in docs.

**Comment by Norserium** [comment, marked as answer]
@trpx, do you want to restrict the image zoom only and preserve the possibility to resize stencil without any limit?

**Reply by trpx** [reply]
Indeed, that is the problem, I figured I can do this using the fixed stencil, but unfortunately I need the stencil to be resizable.
For now I am mitigating the problem by zooming back using the cropper's "zoom" method if user zooms too much, but preventing the user zoom beyond desired boundaries completely would be even better.

**Reply by Norserium** [reply]
Alas, there is no possibility to limit only the image zoom now. You can redefine autozoom algorithm, but it may give you some unexpected effects (for example, resizing of the stencil when user can't zoom image anymore).

**Reply by Norserium** [reply]
@trpx, well, I've got an idea of [the workaround](https://codesandbox.io/s/vue-advanced-cropper-zoom-limit-workaround-j3m81?file=/src/App.vue).

**Reply by trpx** [reply]
@Norserium, awesome! Thank you very much!

**Reply by Norserium** [reply]
@trpx, you are welcome!
